### PR TITLE
Geojson api

### PR DESCRIPTION
**Pull request by rajones-hntb**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.colorCustomizations": {}
+}
```
Comment (jdenen) [inline .vscode/settings.json]: Yeah I don't think this is supposed to be here.

File: lib/discovery_api_web/views/dataset_geojson_view.ex [hunk 2]
```diff
@@ -0,0 +1,11 @@
+defmodule DiscoveryApiWeb.DatasetGeoJsonView do
+  use DiscoveryApiWeb, :view
+
+  def render("features.json", %{features: features, dataset_name: name}) do
```
Comment (bbrewer-papa-x) [inline lib/discovery_api_web/views/dataset_geojson_view.ex]: Nice! Sometimes I forget that we have actual views in Phoenix

File: .vscode/settings.json (hunk 1 above)
Comment (bbrewer-papa-x) [reply .vscode/settings.json]: Yeah, this

Comment (cameronmarsh) [reply .vscode/settings.json]: yeah that's my bad

**Pull request closed without merging**